proto/src/determined/api/v1/job.proto

```diff
@@ -12,8 +12,10 @@ message GetJobsRequest {
   option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
     json_schema: { required: [] }
   };
-  // Request to paginate the resposne.
-  PaginationRequest pagination = 1;
+  // Pagination offset.
+  int32 offset = 10;
+  // Pagination limit.
+  int32 limit = 11;
```

Comment:
what is the philosophical reason for this change when a `PaginationRequest` could enforce consistency of the API?

Reply:
better ux?

Reply:
all of the API is using top-level limit/offset, except for this one `GetJobs` API call. 
we can debate if nested `PaginationRequest` object is good for API v2. for v1 I'd rather normalize to the more popular and widespread option.

Reply:
we seem to use the remaining `Pagination` message in a lot of our APIs. We seem to use them about the same time?

```
hrz ➜ proto git:(master) ag 'Pagination.*=' | wc -l
19
hrz ➜ proto git:(master) ag 'limit =' | wc -l
22
```
currently `[Pagination](https://github.com/hamidzr/determined/blob/60b29812654202a38dacdab0ef3f945ebb02d434/proto/src/determined/api/v1/pagination.proto#L16)` in concept extends `PaginationRequest` . an alternative could be to breakup `Pagination` so that we properly link and enforece PaginationRequest

Reply:
proto3 doesn't have any sort of inheritance, "extend" was a thing in proto2 but it's not that useful. I don't see how we can "reuse" the top level fields. 
member message like `PaginationRequest` is a solution but it'd break everything. I don't think it's worth doing that in v1.